languages/go/example/example.go

```diff
@@ -16,14 +16,15 @@ func main() {
 	bitwardenClient, _ := sdk.NewBitwardenClient(&apiURL, &identityURL)
 
 	accessToken := os.Getenv("ACCESS_TOKEN")
+	statePath := os.Getenv("STATE_PATH")
 	organizationIDStr := os.Getenv("ORGANIZATION_ID")
 	projectName := os.Getenv("PROJECT_NAME")
 
 	if projectName == "" {
 		projectName = "NewTestProject" // default value
 	}
 
-	err := bitwardenClient.AccessTokenLogin(accessToken)
+	err := bitwardenClient.AccessTokenLogin(accessToken, &statePath)
```

Comment:
When not using state, is the idea to pass `nil` or a pointer to an empty string? 

```
err := bitwardenClient.AccessTokenLogin(accessToken, nil)
```

or

```
statePath := ""
err := bitwardenClient.AccessTokenLogin(accessToken, &statePath)
```

Does it work the same way regardless?

Reply:
For a developer using this function, the end result is the same either way. The correct thing to do is pass `nil`, as the underlying function in Rust takes an `Option`, and `nil` maps to the variant `None`. However, in the case an empty string is passed, the SDK won't be able to read or write to a state file, so it ends up being the same thing.